setup.py

```diff
@@ -376,6 +381,37 @@ def run(self):
             call_subprocess(call_arguments)
 
 
+class InstallCompiledPyYaml(Command):
+
+    description = 'Install PyYAML on Windows'
+
+    def initialize_options(self):
+        pass
+
+    def finalize_options(self):
+        pass
+
+    def run(self):
+        if getattr(self.distribution, 'salt_installing_pyyaml_windows', None) is None:
+            print('This command is not meant to be called on it\'s own')
+            exit(1)
+        import platform
+        from pip.utils import call_subprocess
+        from pip.utils.logging import indent_log
+        platform_bits, _ = platform.architecture()
+        call_arguments = ['easy_install', '-Z']
+        if platform_bits == '64bit':
+            call_arguments.append(
+                'http://repo.saltstack.com/windows/dependencies/64/PyYAML-3.11.win-amd64-py2.7.exe'
+            )
+        else:
+            call_arguments.append(
+                'http://repo.saltstack.com/windows/dependencies/32/PyYAML-3.11.win-amd64-py2.7.exe'
```

Comment:
Unless there is a compelling reason not to do so, I would vastly prefer that all downloads take place over a secure connection.